tox.ini

```diff
@@ -30,12 +31,12 @@ whitelist_externals =
     /usr/bin/shellcheck
     /bin/bash
 basepython =
-    {check,doc,doc_travis,doc_travis_deploy}: python3
+    {check,pdfdoc,doc,doc_travis,doc_travis_deploy}: python3
     unit_py3_6: python3.6
     unit_py3_4: python3.4
```

Comment:
Just curiosity: No Python 3.5?

Reply:
yes it seems it never was part of the distro we packaged it for. There was 3.4 and next I saw 3.6
If this is an issue we should resolve please open an extra issue for it such that we don't mix things here. Thanks